eq-author-api/migrations/updateMetadataValue.test.js

```diff
@@ -0,0 +1,73 @@
+const updateMetadataValue = require("./updateMetadataValue.js");
+
+describe("updateMetadataValue", () => {
+  it("should update metadata to correct new format", () => {
+    const metadata = [
+      {
+        alias: "text",
+        dateValue: null,
+        id: "296",
+        key: "ru_ref",
+        languageValue: null,
+        regionValue: null,
+        textValue: "old Value",
+        type: "Text",
+        value: "new Value",
+      },
+      {
+        alias: "date",
+        dateValue: "2019-07-01",
+        id: "294",
+        key: "date",
+        languageValue: null,
+        regionValue: null,
+        textValue: null,
+        type: "Date",
+        value: "2019-04-09",
+      },
+      {
+        alias: "region",
+        dateValue: null,
+        id: "297",
+        key: "region",
+        languageValue: null,
+        regionValue: "GB_GBN",
+        textValue: null,
+        type: "Region",
+        value: "GB_ENG",
+      },
+      {
+        alias: "language",
+        dateValue: null,
+        id: "297",
+        key: "language",
+        languageValue: "cy",
+        regionValue: null,
+        textValue: null,
+        type: "Language",
+        value: "en",
+      },
+      {
+        alias: "language",
```

Comment:
```suggestion
        alias: "region",
```

Reply:
This metadata was intended to be language rather than region, I just forgot to change the type, I've changed it now though to bring it in line with the alias and key.